recipe/meta.yaml

```diff
@@ -167,13 +167,14 @@ requirements:
     - libxcb                             # [linux]
     - {{ pin_compatible("libclang") }}
     - openssl  # exact pin handled through openssl run_exports
-    - mysql 5.7.*  # [unix]
+    - {{ pin_compatible("mysql", max_pin="x.x") }}  # [unix]
     - sqlite
     - libcups 2.*  # [linux]
     - krb5  # exact pin handled through krb5 run_exports
   run_constrained:
     - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.14") }}  # [osx and x86_64]
     - qt >=5.15.2,<6
+    - {{ pin_compatible('postgresql', max_pin='x.x.x') }}
```

Comment:
er I think this might be enforced through libpq anyway?

Reply:
Do you even need postgresql? I think only libpq is needed.

Reply:
Correct, only `libpq` needed for build.

Reply:
then you can remove the run_constrained entry for postgresql, becaue it's enforced via libpq. if you want.

Reply:
Yup, it's in the next commit. Missed it on this one amongst some local changes I was playing with but ultimately didn't keep.